src/lib.rs

```diff
@@ -13,6 +13,7 @@ extern crate serde_derive;
 mod internal {
 	#![allow(dead_code)]
 	include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+	pub use self::root::*;
```

Comment:
I'm not particularly great with Rust modules, can you tell me what this is doing exactly? I'm not sure how a `pub use` works inside of a private module, and am not entirely sure what module `self::root` refers to.

Reply:
This is due to the bindgen changes. Building with c++ namespaces puts all externals inside a module called root.

Reply:
Ahh okay, thanks for the explanation!